src/main/fc/runtime_config.c

```diff
@@ -15,10 +15,16 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// FIXME this file has started to resemble a garbage bin.  move everything out into appropriate locations.
+
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
```

Comment:
we discussed this briefly here, just picking up that ball again - I think it's actually hiding the dependencies since we have to include the whole world, including irrellevant stuff which makes for really annoying extra work when writing tests. (see https://github.com/cleanflight/cleanflight/issues/2191#issuecomment-227884242 and a few comments below that)